tests_integration_with_local_chain/markets/seer/test_seer_market_resolution.py

```
@@ -0,0 +1,34 @@
+
+
+def test_seer_scalar_redemption(
+    seer_subgraph_handler_test: SeerSubgraphHandler,
+) -> None:
+    # ToDO - fork at specific block 41701049
+    # ToDo - mock below (used inside redeem_winnings)
+    """
+    def redeem_winnings(api_keys: APIKeys) -> None:
+        web3 = RPCConfig().get_web3()
+    """
+    w3 = Web3(
```

Ruff (0.12.2)

Local variable `w3` is assigned to but never used

Remove assignment to unused variable `w3`

(F841)